pdr_backend/util/csvs.py

```diff
@@ -1,28 +1,62 @@
 import os
 import csv
+from typing import List, Dict
```

Comment:
In this file, please make all the names of functions follow the same patterns as in df-py. Eg https://github.com/oceanprotocol/df-py/blob/main/df_py/volume/csvs.py. 

This reduces cognitive load as we go from repo to repo.

Reply:
changed function names